src/wp-content/themes/twentytwentyfour/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Twenty Twenty-Four
+Theme URI: https://wordpress.org/themes/twentytwentyfour
+Author: the WordPress team
+Author URI: https://wordpress.org
+Description: Twenty Twenty-Four is designed to be flexible, versatile and applicable to any website. Its collection of templates and patterns tailor to different needs, such as presenting a business, blogging and writing or showcasing work. A multitude of possibilities open up with just a few adjustments to color and typography. Twenty Twenty-Four comes with style variations and full page designs to help speed up the site building process, is fully compatible with the site editor, and takes advantage of new design tools introduced in WordPress 6.4.
+Requires at least: 6.0
```

Comment:
Can we confirm that this is accurate? Twenty Twenty-Three requires at least 6.1. Seems strange to have a newer theme support an older version than the previous one.

Reply:
I can confirm this has yet to be tested, I'll update